package.json

```diff
@@ -64,11 +70,13 @@
   ],
   "repository": "mozilla-b2g/mozilla-raptor",
   "scripts": {
-    "lint": "gulp lint"
+    "lint": "gulp lint",
+    "postinstall": "gulp build",
+    "build": "gulp build"
   },
   "config": {
     "ghooks": {
-      "pre-commit": "npm run lint"
+      "pre-commit": "gulp lint && gulp build"
```

Comment:
Same as previous, gulp isn't installed globally so there is no `gulp` command everywhere. `npm` command is reliable so should use `npm run lint && npm run build`.